src/C# NPC List Refresher Source Code/Program.cs

```diff
@@ -0,0 +1,198 @@
+﻿using HtmlAgilityPack;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Freewar_Converter
+{
+    internal class Program
+    {
+        static List<Data> dataList = new List<Data>();
+
+        public static void Resistenz()
+        {
+            var web = new HtmlWeb();
+            var doc = web.Load("https://www.fwwiki.de/index.php/Resistenz-NPCs_(Liste)");
+
+            var trElements = doc.DocumentNode.Descendants("tr")
+            .Where(x => x.Descendants("td").FirstOrDefault()?.Attributes["style"] != null &&
+            x.Descendants("td").FirstOrDefault()?.Attributes["style"].Value == "text-align:left;");
+
+            foreach (var trElement in trElements)
+            {
+                var tdElements = trElement.Descendants("td").ToList();
+                var aElement = tdElements[0].Descendants("a").FirstOrDefault();
+
+                if (aElement != null)
+                {
+                    string name = aElement.InnerHtml;
+                    string strength = tdElements[1].InnerHtml;
+                    string health = tdElements[2].InnerHtml;
+                    string category = tdElements[8].InnerHtml.Replace("\n", "");

```

Comment:
NIT: kinda fragile way to match. but i guess its fine for now 🤷

Reply:
this is good for now, if its not working anymore (what i dont think because the table is the same since year) then mark me, i fix it ;) or i fix self if i see it in the future